ui/pages/settings/experimental-tab/experimental-tab.component.tsx

```diff
@@ -349,6 +385,7 @@ export default class ExperimentalTab extends PureComponent<ExperimentalTabProps>
           this.renderKeyringSnapsToggle()
           ///: END:ONLY_INCLUDE_IF
         }
+        {this.renderWatchAccountToggle()}
```

Comment:
are we not putting this feature behind a code fence first?

Reply:
@AlexJupiter said we will put it behind an experimental toggle, but no code fencing required 😌